src/NuGet.Core/NuGet.Build.Tasks.Pack/NuGet.Build.Tasks.Pack.targets

```diff
@@ -282,8 +284,8 @@ Copyright (c) .NET Foundation. All rights reserved.
       <PackProjectInputFile>$(MSBuildProjectFullPath)</PackProjectInputFile>
     </PropertyGroup>
   </Target>
-  
-  <Target Name="_GetProjectReferenceVersions" 
+
```

Comment:
Remove the space comments, since it adds no value to the changeset.

Reply:
It's fine but every time I open/save the file, it removes the leading and tailing whitespaces...

Reply:
Will do clean up at the end.

Reply:
Done.